GVFS/GVFS.PlatformLoader/PlatformLoader.Mac.cs

```diff
@@ -1,17 +1,10 @@
 using GVFS.Common;
-using GVFS.Common.Git;
 using GVFS.Platform.Mac;
-using GVFS.Virtualization.FileSystem;
 
 namespace GVFS.PlatformLoader
 {
-    public static class GVFSPlatformLoader
+    public static partial class GVFSPlatformLoader
     {
-        public static FileSystemVirtualizer CreateFileSystemVirtualizer(GVFSContext context, GVFSGitObjects gitObjects)

```

Comment:
Are the `PlatformLoader` changes still needed (now that FastFetch has the native calls for both platforms in its own project)?